contrib/babelfishpg_common/src/datetime.h

```diff
@@ -8,6 +8,10 @@
 #ifndef PLTSQL_DATETIME_H
 #define PLTSQL_DATETIME_H
 
+#include "postgres.h"
+
+#include "datatype/timestamp.h"
```

Comment:
avoid including headers in headers, rather include in C files wherever needed (OK for cases where header file itself needs the def)

Reply:
It is needed in the header file itself, there is a reference for `Timestamp` in this file.